ui/src/buckets/components/BucketList.tsx

```diff
@@ -94,9 +95,11 @@ class BucketList extends PureComponent<Props & WithRouterProps, State> {
               testID="retention-sorter"
             />
           </ResourceList.Header>
-          <ResourceList.Body emptyState={emptyState}>
-            {this.listBuckets}
-          </ResourceList.Body>
+          <GetResources resources={[ResourceType.Labels]}>
```

Comment:
can we put this next to where buckets are fetched?

Reply:
@drdelambre when I did, the empty state was never fulfilled, so I moved it out one layer (by empty state, I mean there's a message that's typically displayed when there are no buckets output in a search)

Reply:
here i think:
https://github.com/influxdata/influxdb/blob/a4e3dce135b5a205d9b51abe89928e0c07e20af0/ui/src/buckets/containers/BucketsIndex.tsx#L62

Reply:
Totally, good eye